File: Day 048/homework/code/codewars2.py
"""https://www.codewars.com/kata/57a2013acf1fa5bfc4000921/train/python"""
def find_average(numbers):
    total = 0
    count = 0
    
    for num in numbers:
        total += num
        count +=1
        
    if count == 0:
        return 0
    return total/count

File: day 84/classwork/code/codewars2.js
// https://www.codewars.com/kata/5626b561280a42ecc50000d1/train/javascript

function sumDigPow(a, b) {
  let result = [];

  for (let num = a; num <= b; num++) {
    let str = String(num);
    let sum = 0;

    for (let i = 0; i < str.length; i++) {
      let digit = Number(str[i]);
      sum += digit ** (i + 1); 
    }

    if (sum === num) result.push(num);
  }

  return result;
}
 